openapi/index.yaml

```diff
@@ -756,6 +755,8 @@ components:
           type: string
           description: Denominazione paese estero
       description: Indirizzo fisico scoperto durante fase di consegna
+    NotificationAttachmentDownloadMetadataResponse:
```

Comment:
Can we create a separate PR with these changes, so I can use them in the feature I'm going to develop?

Reply:
Done! https://github.com/pagopa/pn-local-emulator/pull/47